src/index.mako.html

```diff
@@ -822,15 +806,15 @@
           gaLayersProvider.wmtsGetTileUrlTemplate = '//wmts{s}.geo.admin.ch/1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{y}/{x}.{Format}';
           gaLayersProvider.wmtsMapProxyGetTileUrlTemplate =  gaGlobalOptions.mapproxyUrl + '/1.0.0/{Layer}/default/{Time}/{TileMatrixSet}/{z}/{x}/{y}.{Format}';
           gaLayersProvider.terrainTileUrlTemplate = '//3d.geo.admin.ch/1.0.0/{Layer}/default/{Time}/4326';
-          if (apiEnv || apiParam){
+          if (apiOverwrite){
```

Comment:
I have a hunch that something is wrong with this. When we don't specify anything in the URL as parameters, the layersConfig _should_ come from the cached resource (under same domain). Only when we overwrite the api_url via parameter should this be used for the layersConfig. This is confirmed with tests on current integration, where we acutally call the API and not use the cached version under same URL.
This is important for the offline.

Same is true for the topics URL.